readme.md

```
@@ -48,12 +48,93 @@ The primary configuration format. JSONNet is fully compatible with plain JSON.
 ]
 ```
 
+#### YAML (.yaml, .yml)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)